packages/table/src/constants.js

```diff
@@ -1,4 +1,6 @@
 export const rowHeights = {
   SMALL: "64px",
-  MEDIUM: "80px"
+  MEDIUM: "80px",
 };
+
+export default rowHeights;
```

Comment:
If we're gonna export `rowHeights` as default, we should rename the file.

I think this file was supposed to have other types of constants. So, how about we export an object containing `rowHeights`? This way we won't have breaking changes if we need to add another group of constants in the future.
```suggestion
export default { rowHeights };
```